app/Config/Format.php

```diff
@@ -39,33 +41,4 @@ class Format extends BaseConfig
 		'application/xml'  => \CodeIgniter\Format\XMLFormatter::class,
 		'text/xml'         => \CodeIgniter\Format\XMLFormatter::class,
 	];
-
-	//--------------------------------------------------------------------
-
-	/**
-	 * A Factory method to return the appropriate formatter for the given mime type.
-	 *
-	 * @param string $mime
-	 *
-	 * @return \CodeIgniter\Format\FormatterInterface
-	 */
-	public function getFormatter(string $mime)
```

Comment:
I didn't remove it completely i just relocated it in system/Format/Format.php under CodeIgniter\Format namespace and now all Format instances runs throw \Config\Services::format()

about tests I actually don't know how to do it on the right way, may i get help or something

Reply:
Yes, I'm aware that you moved it to the `Format` class, but it doesn't change the fact that some people might use this method already in their code (just like we do in some of our tests). If we remove this method from here it will break their code - it has to stay.

Reply:
I got your point, but i think we can solve this problem by hinting new changes in next version changelog or something like that, any ideas?

Reply:
No, sorry. It has to stay - it was a public method. Removing it will be a breaking compatibility change.